Update README docs (#89)
ballerina-platform/module-ballerina-log: README.md

This package provides a basic API for logging.

### Log record

> A log record contains the timestamp, log level, module name, and the log message.

Reviewer: ```suggestion
A log record contains the timestamp, log level, module name, and log message.
```

> Users can pass any number of key/value pairs which needs to be displayed in the log message.

Reviewer: ```suggestion
You can pass any number of key/value pairs, which need to be displayed in the log message.
```

> These can be of `anydata` type including int, string and boolean.

Reviewer: ```suggestion
These can be of `anydata` type including int, string, and boolean.
```
